src/sdk-reference/js/6/getting-started/node-js/snippets/create.js

```diff
@@ -0,0 +1,34 @@
+// load the Kuzzle SDK module
+const { Kuzzle } = require('kuzzle-sdk');
+
+// instantiate a Kuzzle client
+// replace 'kuzzle' with your Kuzzle backend hostname, probaly 'localhost'
+const kuzzle = new Kuzzle('websocket', { host: 'kuzzle' });
```

Comment:
Shouldn't we instantiate a `WebSocket` object and pass it to the Kuzzle constructor instead, since using a string is deprecated?

Reply:
We didn't expose the Websocket protocol in the module export for now